courses/ARTH_401.json

```diff
@@ -17,7 +17,7 @@
         "course_components": {
             "Seminar": "Required"
         }, 
-        "enrollment_requirement": "Prerequisites\u00a0\u00a0\u00a0 A GPA of 2.6 in a minium of 24.0 units in ARTH and Level 4 and registration in an ARTH Major or Medial Plan.", 
+        "enrollment_requirement": "Prerequisites\u00a0\u00a0\u00a0 A GPA of 2.6 in 24.0 units in\u00a0 ARTH and Level 4 and registration in an ARTH Major or Medial Plan.", 
```

Comment:
This new character is a non-breaking space. I'm not sure if we want to persist those in our data - maybe consider throwing them out?